package.json

```diff
@@ -456,6 +456,11 @@
 				"category": "GitHub Pull Requests",
 				"icon": "$(globe)"
 			},
+			{
+				"command": "pr.openAllChanges",
+				"title": "Open All Changes",
```

Comment:
What do you think about making this command "Open All Diffs"? I usually prefer to view the diffs for each file, and I think the original issue was also interested in diffs.

Reply:
Oh, yeah 🤦. I'll refactor 👍